File: Wrap-up/Javascript/Factorial.js
const scanner = require("prompt-sync")();

console.log("Get Factorial of Number");

let userNumber = Number(scanner("Enter Number: "));

console.log();

let factorial = 1;

for(let index = userNumber; index > 0; index--){
    factorial *= index;
}

console.log("Factorial of " + userNumber + " -> " + factorial);

File: Wrap-up/Java/Factorial.java
import java.util.Scanner;

public class Factorial {
    public static void main(String... args){
        Scanner input = new Scanner(System.in);
        
        IO.println("Get Factorial of Number");
        
        IO.println("Enter Number: ");
        int userNumber = input.nextInt();
        
        IO.println();
        
        int factorial = 1;
        
        for(int index = userNumber; index > 0; index--){
            factorial *= index;
        }
        
        IO.println("Factorial of " + userNumber + " -> " + factorial);
    }
}
